### PR TITLE
Switch to postgres

### PR DESCRIPTION
**Pull request by MancunianSam**

### CODE REVIEW

File: Jenkinsfile-publish [hunk 1]
```diff
@@ -19,30 +19,25 @@ pipeline {
                         script {
                             def accountNumber = getAccountNumberFromStage()
                             sh "python3 /invoke_lambda.py ${accountNumber} ${params.STAGE} tdr-database-migrations-${params.STAGE}"
-                            slackSend color: "good", message: "*Database migrations* :arrow_up: The migration lambda has been run in *${params.STAGE}*", channel: "#tdr-releases"
+                            //slackSend color: "good", message: "*Database migrations* :arrow_up: The migration lambda has been run in *${params.STAGE}*", channel: "#tdr-releases"
```
Comment (TomJKing) [inline Jenkinsfile-publish:22]: Should this be commented out?

File: Jenkinsfile-publish [hunk 2]
```diff
@@ -19,30 +19,25 @@ pipeline {
...
-                                    sh 'git config --global user.name tna-digital-archiving-jenkins'
-                                    sh "sbt flywayMigrate slickCodegen 'release with-defaults'"
-                                    slackSend color: "good", message: "*Database schema* :arrow_up: The database package has been published", channel: "#tdr-releases"
-                                }
+                            sshagent(['github-jenkins']) {
+                                sh "git push --set-upstream origin ${env.GIT_LOCAL_BRANCH}"
+                                sh 'git config --global user.email tna-digital-archiving-jenkins@nationalarchives.gov.uk'
+                                sh 'git config --global user.name tna-digital-archiving-jenkins'
+                                sh "sbt flywayMigrate slickCodegen 'release with-defaults'"
+                                //slackSend color: "good", message: "*Database schema* :arrow_up: The database package has been published", channel: "#tdr-releases"
```
Comment (TomJKing) [inline Jenkinsfile-publish:40]: Should this be commented out?

File: lambda/src/main/resources/db/migration/V4__transfer_agreement.sql [hunk 3]
```diff
@@ -1,11 +1,16 @@
...
-    AllPublicRecords BOOLEAN DEFAULT NULL,
-    AllCrownCopyright BOOLEAN DEFAULT NULL,
-    AllEnglish BOOLEAN DEFAULT NULL,
-    AllDigital BOOLEAN DEFAULT NULL,
-    AppraisalSelectionSignedOff BOOLEAN DEFAULT NULL,
-    SensitivityReviewSignedOff BOOLEAN DEFAULT NULL,
-    PRIMARY KEY (ConsignmentId),
-    CONSTRAINT ConsignmentId FOREIGN KEY (ConsignmentId) REFERENCES Consignment (ConsignmentId)
-)
+CREATE TABLE "TransferAgreement" (
```
Comment (TomJKing) [inline lambda/src/main/resources/db/migration/V4__transfer_agreement.sql:1]: Should this migration just be included in the new initial migration script?

File: Jenkinsfile-publish (hunk 2 above)
Comment (MancunianSam) [reply Jenkinsfile-publish:40]: No, well spotted. I've changed it.

File: lambda/src/main/resources/db/migration/V4__transfer_agreement.sql (hunk 3 above)
Comment (MancunianSam) [reply lambda/src/main/resources/db/migration/V4__transfer_agreement.sql]: Yeah, you're right, I've moved it.

**Pull request merged**